waiter/src/waiter/utils.clj

```diff
@@ -154,10 +154,10 @@
 
 (defn map->json-response
   "Convert the input data into a json response."
-  [data-map & {:keys [status] :or {status 200}}]
+  [data-map & {:keys [headers status] :or {headers {} status 200}}]
```

Comment:
No longer needed after address my comments above.

Reply:
BTW I saw a pattern I liked.  Not that this needs to change, but:

<pre>
(-> (map->json-response {})
    (rr/header "x-header" "value")
    (rr/status 403))
</pre>
 
Instead of pushing everything we need into this function, just wrap it.  Had to get it off my chest.